repository_service_tuf_api/api/config.py

```diff
@@ -16,6 +16,22 @@
 )
 
 
+@router.put(
+    "/",
+    summary=(
+        "Update settings. " f"Scope: {SCOPES_NAMES.write_settings.value}"
+    ),
+    description="Update configuration settings",
+    response_model=config.PutResponse,
+    response_model_exclude_none=True,
```

Comment:
The default status code is 200 (`OK`.
It is async, sending the task to the broker, it should return `ACCEPTED` 202 as the status code as default.